external/concealer/src/autoware_auto.cpp

```diff
@@ -77,9 +77,8 @@ double AutowareAuto::getSteeringAngle() const
 
 double AutowareAuto::getGearSign() const
 {
-  return getVehicleStateCommand().gear == autoware_auto_msgs::msg::VehicleStateReport::GEAR_REVERSE
-           ? -1.0
-           : +1.0;
+  using AUTOWARE_AUTO_VEHICLE_MSGS::msg::VehicleStateReport;
+  return getVehicleStateCommand().gear == VehicleStateReport::GEAR_REVERSE ? -1.0 : +1.0;
```

Comment:
```
getGearCommand().command == GearCommand::REVERSE
```

This is not an assignment.

Reply:
Sorry for my mistake, but what is gear sign?
Is it gear position of something?
Is it a continuous value?

Reply:
In order to tell the vehicle model whether it is reverse gear or not (= forward or backward), we need to get a positive or negative sign instead of an enum.
The numerical value (= +1.0 or -1.0) obtained from getGearSign is used as the control input of the vehicle model and is used for multiplication.